Real-Time Object Tracker/short.py

```
@@ -0,0 +1,248 @@
+class Sort(object):
+    def __init__(self, max_age=1, min_hits=3, iou_threshold=0.3):
+        """
+        Sets key parameters for SORT
+        """
+        self.max_age = max_age
+        self.min_hits = min_hits
+        self.iou_threshold = iou_threshold
+        self.trackers = []
+        self.frame_count = 0
+
+    def update(self, dets=np.empty((0, 5))):
```

Ruff (0.11.9)

Do not perform function call `np.empty` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)